minigalaxy/window/window.py

```diff
@@ -44,6 +44,11 @@ def __init__(self, name):
         if not os.path.exists(THUMBNAIL_DIR):
             os.makedirs(THUMBNAIL_DIR)
 
+        # Check if the user unset to stay logged in Preference and refresh token
+        if not self.api.config.get("stay_logged_in"):
```

Comment:
This part causes a problem for old installations. An old configuration file will return ``None`` here, but it should be ``True``. Try this code:
```
stay_logged_in = self.api.config.get("stay_logged_in")
        if stay_logged_in is not None and not stay_logged_in:
            self.config.unset("username")
            self.config.unset("refresh_token")
        else:
            self.config.set("stay_logged_in", True)
```